### ISSUE TITLE
cannot use plan (type *terraform.Plan) as type *plans.Changes in argument to format.NewPlan

### ISSUE THREAD

**Issue by yujunz** [afeld/tfmv#7]
Error while installing

```
 ✗ go get github.com/afeld/tfmv
# github.com/afeld/tfmv
../../../go/src/github.com/afeld/tfmv/tfmv.go:28:24: cannot use plan (type *terraform.Plan) as type *plans.Changes in argument to format.NewPlan
```

**Comment by yujunz** [comment]
```
✗ go version
go version go1.11.1 darwin/amd64
```

**Comment by yujunz** [comment]
Resolved by down grading go to 1.9
